r2/r2/models/link.py

```diff
@@ -408,9 +408,6 @@ def slug():
             res = "/%s" % p
         return res
 
-    def make_permalink_slow(self):
-        return self.make_permalink(self.subreddit_slow)
-
```

Comment:
So are most people who work in sane, well-organized, well-documented codebases :-S